### DISCUSSION TITLE
Suggestions on Web Hosting?

### DISCUSSION THREAD

**Discussion by mattWalters0** [supabase/supabase#3579] [Questions] [answered]
Any suggestions on where to host the web front end for my Superbase App?  Need something easy and low maintenance.

Firebase used to handle this as well.

**Comment by zakaria-chahboun** [comment]
https://render.com/ is my favourite at this moment!

**Comment by MildTomato** [comment, marked as answer]
This thread would not be complete without at least mentioning [Vercel](https://vercel.com/docs/concepts/deployments/overview) or [Netlify](https://docs.netlify.com/site-deploys/create-deploys/).

**Comment by mattWalters0** [comment]
Thanks for the suggestions @zakaria-chahboun @MildTomato !
